src/tutorials/pet-shop.md

```diff
@@ -277,7 +277,7 @@ Now we are ready to create our own migration script.
 You've now written your first smart contract and deployed it to a locally running blockchain. It's time to interact with our smart contract now to make sure it does what we want.
 
 
-## Testing the smart contract
+## Testing the smart contract using Solidity
 
 Truffle is very flexible when it comes to smart contract testing, in that tests can be written either in JavaScript or Solidity. In this tutorial, we'll be writing our tests in Solidity.
```

Comment:
Perhaps "in this tutorial" should now be changed to "in this section, we'll write our tests in Solidity."